README.md

```diff
@@ -1,5 +1,13 @@
 ## sbt-xjc An SBT Plugin to compile XML Schemata to JAXB Java sources with XJC
 
+[JAXB - Java Architecture for XML Binding](https://javaee.github.io/jaxb-v2/)
+
+[XJC documentation](https://javaee.github.io/jaxb-v2/doc/user-guide/ch04.html#tools-xjc)
+
+XJC plugins
```

Comment:
there's not a lot of context here. why are these relevant to users of this project?

Reply:
Because this plugin calls the XJC tool to do its work. Command line arguments are passed through to XJC.

Reply:
There's no documentation on the jaxb2-commons repo. I feel that linking to it with no documentation would be very confusing for our users

There's documentation for the second project: https://mklemm.github.io/jaxb2-rich-contract-plugin/usage.html. However, this documentation is all for a Maven project. I feel that if we're going to reference it from this project we should show how you would use it with sbt-xjc

Reply:
Yes, the Java EE documentation seems to be in a very bad state since it move to github. 
I found those links helpful while re-configuring our build with sbt-xjc.